README.md

```diff
@@ -68,11 +68,14 @@ and health-checking mechanisms.
 
   * .. or from sources
     ```sh
+    
```

Comment:
probably this additional line needed to be removed (line 71)

Reply:
Done!